library/eventstore

```diff
@@ -0,0 +1,7 @@
+Maintainers: EventStore Team <info@eventstore.org> (@eventstore)
+
+GitRepo: https://github.com/EventStore/eventstore-docker
+
+Tags: release-4.1.1-hotfix1
+GitCommit: cd0eafec71baffbcc0b9a4bb7c58e996cf290d34
```

Comment:
I cannot find that commit in https://github.com/EventStore/eventstore-docker.

Reply:
Yeah, it appears this is a commit on the main EventStore repo, not the eventstore-docker repo: https://github.com/EventStore/eventstore/commit/cd0eafec71baffbcc0b9a4bb7c58e996cf290d34

I'd recommend giving https://github.com/docker-library/official-images#contributing-to-the-standard-library a read-through (and @TimWolla's other comments were definitely spot-on :heart:).  There appear to be quite a few things noted in the docs there that aren't adhered to here.

I'd also add that `apt-get autoremove` isn't likely to do anything (since you don't have any packages being _removed_ here), and `apt-get clean` is done automatically by the base image itself.  Also, if your install process is leaving extra files in `/tmp` or `/var/tmp`, I'd highly recommend trying to figure out why (since installing a Debian package shouldn't typically be leaving trash in those places).